community/abcde/APKBUILD

```diff
@@ -1,12 +1,12 @@
 # Maintainer:
 pkgname=abcde
-pkgver=2.9.2
+pkgver=2.9.3
 pkgrel=0
 pkgdesc="A command line CD encoder that reads your CD, downloads the track information from a CDDB provider, and rips your CD"
 url="http://abcde.einval.com/"
 arch="noarch"
 license="GPL-2.0"
-depends="bash cd-discid py-eyed3"
+depends="bash cd-discid py3-eyed3"
 depends_dev=""
 makedepends="$depends_dev"
```

Comment:
can be removed as the variable it uses is empty

Reply:
@maxice8 all fixed